source/geod24/concurrency.d

```diff
@@ -973,7 +972,17 @@ protected:
             FiberScheduler.yield();
         }
 
-        private bool notified;
+        bool should_block() nothrow
```

Comment:
We use `camelCase`, not `snake_case` for function names. Also please add a ddoc comment.

Reply:
Old habits :) Noted.